resources/platformsh/ibexa-commerce/4.5.x-dev/.platform.app.yaml

```diff
@@ -144,13 +147,11 @@ hooks:
     build: |
         set -e
 
-        # Install newer version of Node.js, as current default (v6LTS) does not play well with WebpackEncore
-        unset NPM_CONFIG_PREFIX
-        curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.8/install.sh | dash
-        export NVM_DIR="$HOME/.nvm"
-        [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
-        nvm current
-        nvm install 14.18.3
+        # Install the version specified in the .nvmrc file
+        n auto
```

Comment:
We might have to prepend this command with `hash n && `, so that if we're running on DDEV with ddev-platformsh the hook wouldn't fail. The `n` isn't available in DDEV, however DDEV has an ability to select NodeJS version (default one, right now, is `16`, which also works for us).